### ISSUE TITLE
Coala's AGPL coverage 

### ISSUE THREAD

**Issue by briankhsieh** [coala/coala#6045]
Hi Coala maintainers, 

Can you help clarify AGPL coverage in Coala's case? Specifically, are the modifications made by Coala, which would get put into the input source code, covered by AGPL? 

Your clarification will help evaluate the technology selection. I'd be helpful If you could clarify your interpretation in the Q&A page. 

Thanks,

Brian

**Comment by jayvdb** [comment]
Hi @briankhsieh , No, AGPL covers only coala code in the repository, not any generated code / modifications .

And each coala bear has metadata for the license of their own code (usually also AGPL), but most of the bears use external linters which have their own licensing, but all of them are open source.  While some *may* **try** to impose some restrictions on modifications, I am not aware of any external linter used in coala-bears which attempts that.

Note that any attempt to impose restrictions on modifications would be laughable, as the linters are using the input source and making automated changes to them.  By definition, automated changes are not a creative work, and so they do not create a derivative work.

I agree this should be clarified in our docs.

And our bears repo should be audited, to be 100% sure.

And the usual disclaimer that I am not a lawyer and this does not constitute legal advice. ;-)

**Comment by briankhsieh** [comment]
Thank you for the clarification @jayvdb. 

It'd be great to see clarification of the reach of the license being on the public-facing materials. That would help us accelerate the technology adoption decision. 

Let me know if there's anything I can do to help.

**Comment by briankhsieh** [comment]
Hi @jayvdb, 

We understand from your comment on August 3, 2019 that Coala's license covers only Coala code in the repository, not any generated code/modifications to user code, and that there are plans to update the repos documentation.  Let us know if we misunderstood.

**Comment by jayvdb** [comment]
Yes, that is correct.  Sorry that the docs havent been updated yet.  I'll assign myself, and tackle it this week sometime.

**Comment by briankhsieh** [comment]
Thank you for confirming and working on this, @jayvdb.
